napari/layers/points/_tests/test_points.py

```diff
@@ -674,6 +674,16 @@ def test_text_error(properties):
         Points(data, properties=copy(properties), text=123)
 
 
+def test_select_properties_object_dtype():
+    """selecting points when they have a property of object dtype should not fail"""
+    # pandas uses object as dtype for strings by default
```

Comment:
Really useful comment! I think I was confused about where `dtype=object` was coming from in some usage, so I guess this explains that.